po/pl.po

```diff
@@ -0,0 +1,441 @@
+
+#. Translators: section header for mapping one button's click to another.
+#: piper/buttondialog.py:144
+msgid "Button mapping"
+msgstr "Mapowanie przycisku"
+
+#. Translators: section header for assigning special functions to buttons.
+#: piper/buttondialog.py:153
+msgid "Special mapping"
+msgstr "Mapowanie specjalne"
+
+#: piper/buttondialog.py:223 piper/buttonspage.py:82
+msgid "Button {} click"
+msgstr "Kliknięcie przycisku {}"
+
+#: piper/buttonspage.py:86
+msgid "Macro: {}"
+msgstr "Makro: {}"
+
+#. Translators: the button is turned disabled, e.g. off.
+#: piper/buttonspage.py:89
+msgid "Disabled"
+msgstr "Wyłączone"
+
+#. Translators: the button has an unknown function.
+#: piper/buttonspage.py:92 piper/ratbagd.py:613
+msgid "Unknown"
+msgstr "Nieznane"
+
+#: piper/buttonspage.py:100
+msgid "Configure button {}"
+msgstr "Konfiguracja przycisku {}"
+
+#: piper/mouseperspective.py:86 data/ui/ResolutionsPage.ui:209
+msgid "Resolutions"
+msgstr "Rozdzielczości"
+
+#: piper/mouseperspective.py:88
+msgid "Buttons"
+msgstr "Przyciski"
+
+#: piper/mouseperspective.py:90
+msgid "LEDs"
+msgstr "Diody LED"
+
+#: piper/mouseperspective.py:93 piper/profilerow.py:41
+msgid "Profile {}"
+msgstr "{}. profil"
+
+#: piper/ratbagd.py:604
+msgid "Left mouse button click"
+msgstr "Kliknięcie lewym przyciskiem myszy"
+
+#: piper/ratbagd.py:605
+msgid "Right mouse button click"
+msgstr "Kliknięcie prawym przyciskiem myszy"
+
+#: piper/ratbagd.py:606
+msgid "Middle mouse button click"
+msgstr "Kliknięcie środkowym przyciskiem myszy"
+
+#: piper/ratbagd.py:607
+msgid "Backward"
+msgstr "Wstecz"
+
+#: piper/ratbagd.py:608
+msgid "Forward"
+msgstr "Dalej"
+
+#: piper/ratbagd.py:614
+msgid "Doubleclick"
+msgstr "Podwójne kliknięcie"
+
+#: piper/ratbagd.py:615
+msgid "Wheel Left"
+msgstr "Kółko w lewo"
+
+#: piper/ratbagd.py:616
+msgid "Wheel Right"
+msgstr "Kółko w prawo"
+
+#: piper/ratbagd.py:617
+msgid "Wheel Up"
+msgstr "Kółko w górę"
+
+#: piper/ratbagd.py:618
+msgid "Wheel Down"
+msgstr "Kółko w dół"
+
+#: piper/ratbagd.py:619
+msgid "Ratchet Mode"
+msgstr "Tryb zapadkowy"
+
+#: piper/ratbagd.py:620
+msgid "Resolution Cycle Up"
+msgstr "Cykl rozdzielczości w górę"
+
+#: piper/ratbagd.py:621
+msgid "Resolution Cycle Down"
+msgstr "Cykl rozdzielczości w dół"
+
+#: piper/ratbagd.py:622
+msgid "Resolution Up"
+msgstr "Rozdzielczość w górę"
+
+#: piper/ratbagd.py:623
+msgid "Resolution Down"
+msgstr "Rozdzielczość w dół"
+
+#: piper/ratbagd.py:624
+msgid "Resolution Switch"
+msgstr "Przełączenie rozdzielczości"
+
+#: piper/ratbagd.py:625
+msgid "Resolution Default"
+msgstr "Domyślna rozdzielczość"
+
+#: piper/ratbagd.py:626
+msgid "Profile Cycle Up"
+msgstr "Cykl profilu w górę"
+
+#: piper/ratbagd.py:627
+msgid "Profile Cycle Down"
+msgstr "Cykl profilu w dół"
+
+#: piper/ratbagd.py:628
+msgid "Profile Up"
+msgstr "Profil w górę"
+
+#: piper/ratbagd.py:629
+msgid "Profile Down"
+msgstr "Profil w dół"
+
+#: piper/ratbagd.py:630
+msgid "Second Mode"
```

Comment:
AFAIK it's like a shift key, so button X in second mode actually sends button Y events.